src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp

```diff
@@ -327,6 +338,79 @@ void PeHeuristics::getVisualBasicHeuristics()
 	}
 }
 
+int32_t PeHeuristics::get_int32_unaligned(const uint8_t * codePtr)
```

Comment:
Nitpick: In general, in the C++ part of RetDec, we name variables and functions via `camelCase`. Thus, for the sake of consistency, I suggest renaming the function to `getInt32Unaligned()`.